modules/oauth-list-tokens.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-oauth-access-tokens.adoc
+
+[id="oauth-list-tokens_{context}"]
+= Listing user OAuth access tokens
+
+You can list all of your user OAuth access tokens. Token names are not sensitive and cannot be used to log in.
+
+.Procedure
```

Comment:
Please also document the possibility of listing tokens per given oauth client (that's effectively listing them per the application the tokens were issued for)